Makefile

```diff
@@ -13,10 +13,10 @@ CRD_OPTIONS ?= "crd:trivialVersions=true,preserveUnknownFields=false"
 ENVTEST_K8S_VERSION = 1.21
 
 # Get the currently used golang install path (in GOPATH/bin, unless GOBIN is set)
-ifeq (,$(shell go env GOBIN))
-GOBIN=$(shell go env GOPATH)/bin
+ifeq (,$(shell go1.19 env GOBIN))
```

Comment:
I wouldn't expect all the people on the internet to have a go1.19 binary installed. Most people just have `go`, I think. I've never had `go<suffix>` installed, for example. 

But I'm sure this complication must not be a part of the code - if the user has wrong `go` installed and things break, it's his fault, we have clear prerequisites saying `go 1.19+`.

Reply:
revert back to `go`